### PR TITLE
New package: AusweisApp2:1.22.1

### PR DESCRIPTION
**Pull request by Justinizer**
#### General
- [X] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [X] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/AusweisApp2/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'AusweisApp2'
+pkgname=AusweisApp2
+version=1.22.1
+revision=1
+build_style=cmake
+hostmakedepends="pkg-config qt5-qmake qt5-host-tools"
+makedepends="qt5-devel qt5-tools-devel pcsclite-devel qt5-svg-devel qt5-websockets-devel qt5-declarative-devel openssl-devel qt5-quickcontrols2-devel"
```
Comment (ericonr) [inline srcpkgs/AusweisApp2/template:7]: Please break this at ~80 columns

Comment (Piraty) [reply srcpkgs/AusweisApp2/template]: and use a leading whitespace in the new line

Comment (Justinizer) [reply srcpkgs/AusweisApp2/template]: I did break at 80 columns and got a leading tabular in the new line with commit 
6a31d0e

Comment (Piraty) [reply srcpkgs/AusweisApp2/template]: indeed, and no template does that. for style consistency, i asked you to use a space instead of tab.

Comment (Justinizer) [reply srcpkgs/AusweisApp2/template]: now I understand you :-)
Was a little irritated will fix it in 10 minutes

Comment (Justinizer) [reply srcpkgs/AusweisApp2/template]: why I got the xlint error indent with tabs now?

**Review by Piraty — APPROVED** [review]

**Pull request merged**